pennylane/transforms/classical_jacobian.py

```diff
@@ -65,39 +72,126 @@ def classical_jacobian(qnode):
 
     - The zeroth element of ``weights`` is repeated on the first two gates generated by the QNode.
 
-    - The second column consisting of all zeros indicates that the generated quantum circuit does
-      not depend on the first element of ``weights``.
+    - The third row consisting of all zeros indicates that the third gate ``RY(2.5)`` does not
+      depend on the ``weights``.
+
+    - The quadratic dependence of the fourth gate argument yields :math:`2\cdot 0.6=1.2`.
+
+    .. note::
+
+        The QNode is constructed during this operation.
+
+    For a QNode with multiple QNode arguments, the arguments with respect to which the
+    Jacobian is computed can be controlled with the ``argnums`` keyword argument.
+    The output for ``argnums=None`` depends on the backend:
+
+    .. list-table:: Output format of ``classical_jacobian``
+       :widths: 25 25 25 25
+       :header-rows: 1
+
+       * - Interface
+         - `argnums=None`

```

Comment:
Do these need double backticks? Seems to be an issue with the rendering:

![image](https://user-images.githubusercontent.com/2959003/133027881-52da6f9d-104a-41fe-9b75-2c7c527553b2.png)

Reply:
Oh, yes, of course! :)